content/applications/hr/payroll/contracts.rst

```diff
@@ -0,0 +1,250 @@
+  the drop-down menu.
+
+   - :guilabel:`CDI` is an open-ended contract with only a start date but no end date.
+   - :guilabel:`CDD` is a contract with both a start date and an end date.
+   - :guilabel:`PFI` is a Belgian-specific contract used when hiring employees that need training,
+     and covers the training period specifically.
+
+- :guilabel:`End Date`: If the contract has a specific end date, click the drop-down menu, navigate
+  to the correct month and year using the arrow icons, then click on the date.
+- :guilabel:`HR Responsible`: If there is a specific person in HR that is responsible for the
+  contract, select the person from the drop-down menu.
+- :guilabel:`Analytic Account`: This field allows a link between the contract and a specific
+  analytic account for accounting purposes.
+
+Contract details
+----------------
+
+The contract details section allows for the addition and editing of a contract, and the ability to
+send the contract to the employee for approval and signatures.
+
+.. image:: contracts/contract-details.png
+   :align: center
+   :alt: Contract details in optional tabs for a new contract.
+
+- :guilabel:`Contract Template`: Select a pre-existing contract template from the drop-down menu.
+  Contract templates are typically created through the *Recruitment* application.
+- :guilabel:`New Contract Document Template`: Select a contract from the drop-down menu to be
+  modified for this new employee contract.
+- :guilabel:`Contract Update Document Template`: Select a contract from the drop-down menu if the
+  employee has an existing contract that requires updating.
+- :guilabel:`Notes`: The notes field is a text field where any notes for the employee contract can
+  be entered for future reference.
+
+Modifying a contract
+~~~~~~~~~~~~~~~~~~~~
+
+Click the :guilabel:`External Link` button at the end of each line to open the corresponding
+contract template and make any changes.
+
+.. image:: contracts/external-link.png
+   :align: center
+   :alt: Contract details in optional tabs for a new contract.
+
+A pop-up window appears with all the contract details. Modify the fields for the contract as needed.
+
+.. image:: contracts/modify-contract.png
+   :align: center
+   :alt: Edit the details for the contract.
+
+- :guilabel:`Tags`: Select any tags associated with the contract.
+- :guilabel:`Signed Document Workspace`: This is where the signatures are stored. Choose a
+  pre-configured workspace or create a new one.
+- :guilabel:`Signed Document Tags`: Select or create any tags associated only with the signed
+  contract as opposed to the original unsigned contract.
+- :guilabel:`Redirect Link`: Enter a redirect link for the employee to access the contract. A
+  redirect link takes the user from one URL to another, in this case, to the newly updated contract
+  specifically written for them.
+- :guilabel:`Who can Sign`: Select either *All Users* or *On Invitation*.
```

Comment:
Consider making these guilabels, since you're instructing the reader to select options in the UI. But I think it's okay either way.
```suggestion
- :guilabel:`Who can Sign`: Select either :guilabel:`All Users` or :guilabel:`On Invitation`.
```